pages/eventos/editar/[id].tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react'
+import { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { useMutation, useLazyQuery } from '@apollo/client'
+import { GET_EVENT_BY_ID } from '../../../services/GraphQL/queries/events'
+import { UPDATE_EVENT } from '../../../services/GraphQL/mutations/events'
+import { EventBase } from '../../../services/GraphQL/types/events'
+import ClientOnly from '../../../views/Shared/ClientOnly'
+import styles from '../../../styles/Home.module.css'
+import { formatDate } from '../../../services/utils/dateFormat'
+
+const EditEvent: NextPage = () => {
+  const [getOriginalEventData, { loading, data }] = useLazyQuery(GET_EVENT_BY_ID)
+  const { push, query } = useRouter()
+  const [updatedEvent, setUpdatedEvent] = useState<EventBase>({
+    title: data?.title || '',
+    description: data?.description || '',
+    date: data?.date || '',
+    type: data?.type || '',
+  })
+  const [date, setDate] = useState('')
+  const [time, setTime] = useState('')
+
+  const [updateEvent] = useMutation(UPDATE_EVENT)
+
+  useEffect(() => {
+    const getCurrentData = async (id: string) => {
+      const { called, error } = await getOriginalEventData({
+        variables: {
+          id,
+        },
+      })
+
+      if (error || !called) {
+        console.error(error)
+      }
+
+      if (called) {
+        const { event } = data
+        setUpdatedEvent(event)
+        setDate(event.date?.toString().slice(0, 16).split('T')[0] || '')
+        setTime(event.date?.toString().slice(0, 16).split('T')[1] || '')
+      }
+    }
+
+    if (query.id) getCurrentData(query.id as string)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Could we change this to `eslintrc` file?

Reply:
Hmm I am ok with move that rule to eslint, no prob

Reply:
Ok, I'll do it